package.json

```diff
@@ -65,7 +65,8 @@
   "dependencies": {
     "@0xged/hardhat-deploy": "0.11.5",
     "@chainlink/contracts": "0.4.2",
-    "@mean-finance/deterministic-factory": "1.3.2"
+    "@mean-finance/deterministic-factory": "1.3.2",
+    "@openzeppelin/contracts": "4.7.3"
```

Comment:
We didn't mark it as a dependency because it was imported from `mean-finance/deterministic-factory`, but it should have been a dependency